collect_app/src/main/java/org/odk/collect/android/widgets/StringWidget.java

```diff
@@ -40,12 +59,16 @@ protected StringWidget(Context context, QuestionDetails questionDetails, boolean
 
         readOnly = questionDetails.getPrompt().isReadOnly() || readOnlyOverride;
         answerText = getAnswerEditText(readOnly, getFormEntryPrompt());
-        setUpLayout();
+        setUpLayout(context);
     }
 
-    protected void setUpLayout() {
+    protected void setUpLayout(Context context) {
         setDisplayValueFromModel();
-        addAnswerView(answerText);
+
+        Resources resources = context.getResources();
+        int marginStandard = dpFromPx(context, resources.getDimension(R.dimen.margin_standard));
```

Comment:
Perhaps using `getDimensionPixelSize` would make it more obvious that pixels are returned. I remembered you rightfully complained about this method in Slack.

Reply:
That is at least more obvious!